src/lib/mongodb.ts

```diff
@@ -14,22 +14,11 @@ declare global {
   }
 }
 
-const host = process.env.MONGO_HOST || ''
-const user = process.env.MONGO_USER || ''
-const password = process.env.MONGO_PASSWORD || ''
-
-const RDS_TLS_CERT = process.env.RDS_TLS_CERT || 'rds-combined-ca-bundle.pem'
-
 // Connection string for DocumentDB
-const connectionString =
-  process.env.MONGO_URL ||
-  `mongodb://${user}:${password}@${host}/?tls=true&tlsCAFile=${RDS_TLS_CERT}&replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false`
+const connectionString = process.env.MONGO_URL
```

Comment:
I cleaned this up because as far as I can tell, `MONGO_URL` is defined in all environments (included local dev) and so the connection string logic is no longer used. @abbyoung do you mind confirming this looks right to you?

Reply:
looks good to me!

Reply:
reader: it was not